src/component/handlers/edit/editOnBeforeInput.js

```diff
@@ -155,7 +155,7 @@ function editOnBeforeInput(
     const nativeSelection = global.getSelection();
     // Selection is necessarily collapsed at this point due to earlier check.
     if (
-      nativeSelection.anchorNode !== null &&
+      nativeSelection.anchorNode &&
```

Comment:
I generally prefer to be more specific when checking for a value to be defined, but in this case we don't want any other falsy JS values either, so I think this is ok.